etc/fibo-vue/src/views/Courses.vue

```diff
@@ -1,313 +1,96 @@
+</p>
+</div>
+<h1>
+<span>Offer</span>
+</h1>
+<div style="text-align: justify;">
+<p>EDMC offers courses in Knowledge Graph, available in multiple forms:</p>
+<ol>
+<li>Instructor-led remote instruction by one of the EDM Council&rsquo;s certified trainers. These courses are open enrollment, registration at the EDM Council web site,</li>
+<li>Enterprise focused instruction. The same course material and instructor as our open enrollment course, but with the audience limited to knowledge workers from your organization. This allows more specific focus on use cases and industries of interest to your needs.</li>
+<li>Formal ELearning courses (to be available summer, 2020). The EDM Council is developing a Certification Package in Knowledge Graph. This is a deep-dive into the full details of knowledge graph technology and business practice.</li>
+</ol>
+<p>In these challanging times EDMC will shortly issue&nbsp;<em>Open Knowledge graph and FIBO Fundamentals Course</em>.</p>
+<h5><span>Open Knowledge Graph and FIBO Fundamentals Course</span></h5>
+<p>This course will be virtually delivered on two consecutive days, 9:00am &ndash; 3:00pm each day&nbsp;with breaks.</p>
+<p>It will consists of 5 main modules:</p>
+<ul>
+<li>A. Knowledge Graph Business Case and Market Adoption [2:15]
+<ul>
+<li>A1. Knowledge Graph Perspective On Data</li>
+<li>A2. Knowledge Graph Industry Use Cases</li>
+<li>A3. Knowledge Graphs Takeaways</li>
+</ul>
+</li>
+<li>B. Knowledge Graph Technical Framework [3:00]
+<ul>
+<li>B1. Knowledge Graph Enterprise Architecture Overview</li>
+<li>B2B3&B6. Knowledge Graph Data Languages (RDF, RDFS, SKOS)</li>
+<li>B4. Knowledge Graph Modeling Language (OWL)</li>
+<li>B5. Knowledge Graph Query language (SPARQL)</li>
+<li>B8. Knowledge Graph Enterprise Architecture Ecosystem</li>
+</ul>
+</li>
+<li>C. FIBO Industry Model review [2:00]
+<ul>
+<li>C1. Structure</li>
+<li>C2. Content</li>
+<li>C3. Infrastructure</li>
+</ul>
+</li>
+<li>D. Knowledge Graph Hands-on Exercises [2:00]
+<ul>
+<li>D1. Swaps in Finance</li>
+<li>D2. Toxic byproducts in Agriculture</li>
+<li>D3. Automotive manufacturing</li>
+</ul>
+</li>
+<li>E. Knowledge Graph Course Key Takeaways [0:15]
+<ul>
+<li>E1. Survey</li>
+<li>E2. Accreditation</li>
+</ul>
+</li>
+</ul>
+<h1><span>Course Pallete</span></h1>
+<p><em>Open Knowledge graph and FIBO Fundamentals Course</em> is just part of our educational offer. Our full offer of courses covers different areas of Knowledge Graph Industry. These are shown below as a mind map that interlinks different learning topics.</p>

```

Comment:
capital G for Graph

Reply:
@trypuz  Pete is right on this one; you've fixed most of his other typos, but I think you missed this one (or maybe I'm a version or two behind)